tests/scene/test_fontfile.h

```diff
@@ -38,6 +38,71 @@
 
 namespace TestFontfile {
 
+#ifdef MODULE_FREETYPE_ENABLED
+// Macro to generate tests for getters.
+#define GETTER_TEST(m_name, m_getter, m_default_value)                                                                                          \
+	TEST_CASE("[FontFile] Load Dynamic Font - get-set " m_name) {                                                                               \
+		String test_dynamic_font = String("thirdparty").path_join("fonts").path_join("NotoSansHebrew_Regular.woff2");                           \
```

Comment:
When using string literals, you don't need to use `path_join()` as Godot accepts forward slash paths on all platforms:

```suggestion
		String test_dynamic_font = "thirdparty/fonts/NotoSansHebrew_Regular.woff2");                           \
```

Reply:
Changed to using /